### PR TITLE
[Deps] Bump gl-native to v10.3.2

### PR DESCRIPTION
**Pull request by ank27**
### Summary of changes
Bump gl-native to v10.3.2

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [x] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog>Update gl-native to v10.3.2</changelog>`.
 - [x] If this PR is a `v10.3` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.3` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,9 +42,10 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix location indicator layer rendering when SwiftShader is used. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
 * Avoid possible crash at program exit caused by dummy tracer accessed after destruction. ([#1116](https://github.com/mapbox/mapbox-maps-android/pull/1116))
 * Fix crash for the case when a map event is handled by an Observer of a destructed map. ([#1116](https://github.com/mapbox/mapbox-maps-android/pull/1116))
+* Deliver event to observable subscribers even if observable is already destructed. ([#1134](https://github.com/mapbox/mapbox-maps-android/pull/1134))
```
Comment (kiryldz) [inline CHANGELOG.md:45]: IMO we don't need that entry as this was working fine before #1116 and we actually return to previous correct behaviour.

Comment (ank27) [reply CHANGELOG.md:45]: Good point, should we remove the entries from #1116 too, as this is reverted now?

**Review by alexander-kulikovskii — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ank27) [reply CHANGELOG.md]: removed this entry.

**Review by Chaoba — APPROVED** [review]
Should we also including the changelog for gl-native in this pr?

**Review by pengdev — APPROVED** [review]

**Pull request merged**